test/controllers/NCTSMessageControllerSpec.scala

```diff
@@ -173,6 +175,7 @@ class NCTSMessageControllerSpec extends SpecBase with ScalaCheckPropertyChecks w
             .withHeaders("X-Message-Type" -> MessageType.CancellationDecision.code)
 
           val result = route(application, request).value
+          contentAsString(result) mustEqual "Failed to transition from PositiveAcknowledgement to CancellationDecision"
```

Comment:
The generated error message could be changed to the following:<br>`The message status failed to transition from 'PositiveAcknowledgement' to 'CancellationDecision'.`